streets_reader/src/osm_reader/reader.rs

```diff
@@ -182,10 +182,6 @@ fn read_tags(reader: &mut Peekable<ElementReader>) -> Tags {
         let obj = reader.next().unwrap();
         let key = obj.attribute("k");
         let value = obj.attribute("v");
-        // Filter out really useless data
```

Comment:
This was intended as a file size optimization, but it's inappropriate at this layer. That becomes very obvious now!